pkgs/development/compilers/haxe/default.nix

```
@@ -17,7 +42,9 @@ let
         rev = "refs/tags/${version}";
```

afaik `refs/tags/` shouldn't be needed.
```suggestion
        rev = version;
```

---

what was the result here?

---

84bbcc9ba18